history-of-the-quakers/original/09-book-1-chapter-6.adoc

```diff
@@ -1,11 +1,13 @@
 == Chapter VI.
 
-The Society increaseth under their Sufferings.--Sufferings of James
-Nayler.--James Nayler imprisoned for not putting off his Hat,
-and as a Vagabond;
-and Francis Howgill along with him.--James Nayler tried for Blasphemy
-and acquitted.--Note on Nicholson and Burn`'s Extract from Higginson`'s
-Memoirs.--Imprisonment of Thomas Aldam.
+[.chapter-synopsis]
+* The Society increaseth under their Sufferings
+* Sufferings of James Nayler
+* James Nayler imprisoned for not putting off his Hat, and as a Vagabond; and Francis Howgill along with him
+// I thought I remember at one point you guys wanted me to break up these longer synopses? Should I still do that?
```

Comment:
Yeah, Jason may have said that, and I think sometimes it is appropriate.  I would say, if it's really long and there is an obvious spot to break it go for it. By obvious I mean a change of subject, and it should stand on it's own ideally without re-wording.

Now, for this spot, I'd say it's long enough to break up, but it's sort of borderline that the second part would stand alone well if we broke it at the semicolon, because it uses the word "him" to refer back to Nayler. So, I'd say... I think i'd be fine either way here, but I lean slightly towards keeping this one as is.